### ISSUE TITLE
Unable to override indent rule

### ISSUE THREAD

**Issue by litzebauer** [eslint/eslint#3284]
I am trying to extend the Airbnb rules which use "indent": [2, 2].  In my rules I have "indent": 2 for 4 space indents.  Using Eslint version 1.0.0.

#### Errors
* 8:9   error    Expected indentation of 6 characters but found 8  indent
* 15:6   error    Expected indentation of 6 characters but found 4  indent
* 16:2   error    Expected indentation of 2 characters but found 0  indent

#### Source
```javascript
'use strict';

angular.module('sampleProject').controller('sampleCtrl', function($window) {

    var _this = this;

    function getGreeting(name) {
        return `Hi ${name}!`;
    }

    _this.sayHello = function() {
        let message = getGreeting(_this.name);

        $window.alert(message);
    };
});
```

**Comment by eslintbot** [comment]
Thanks for the issue! We get a lot of issues, so this message is automatically posted to each one to help you check that you've included all of the information we need to help you.

Reporting a bug? Please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
1. The source code that caused the problem
1. The configuration you're using (for the rule or your entire config file)
1. The actual ESLint output complete with line numbers

Requesting a new rule? Please be sure to include:

1. The use case for the rule - what is it trying to prevent or flag?
1. Whether the rule is trying to prevent an error or is purely stylistic
1. Why you believe this rule is generic enough to be included

Requesting a feature? Please be sure to include:

1. The problem you want to solve (don't mention the solution)
1. Your take on the correct solution to problem

Including this information in your issue helps us to triage it and get you a response as quickly as possible.

Thanks!

**Comment by litzebauer** [comment]
Changing my rule to "indent": [2, 4] resolved my issue.
